bedrock/firefox/templates/firefox/includes/download-button.html

```diff
@@ -67,6 +67,7 @@
            data-link-type="download"
```

Comment:
@alexgibson one of the requirements is to add `data-link-type="Download Firefox (RU)"` but I have concerns seeing as it is hard-coded to be `"download"`

Considering some sort of `{% if yandex %}data-link-type="Download Firefox (RU)"{% else %}data-link-type="download"{% endif %}` but am a little unsure how to define if it's `yandex`? Am also unsure if changing it to something other than `"download"` breaks it.